src/indexeddb.js

```diff
@@ -90,6 +83,9 @@
   };
 
   IndexedDB.prototype = {
+    /**
+     * @todo Document
```

Comment:
We use `TODO` comments instead of these currently.

Reply:
I'll switch them back to `TODO`